docs/self-managed/setup/deploy/amazon/amazon-eks/terraform-setup.md

```diff
@@ -33,203 +43,195 @@ To try out Camunda 8 or develop against it, consider signing up for our [SaaS of
 
-:::
+  To configure the AWS CLI:
 
-:::warning
+  ```bash
+  aws configure
+  ```
 
-Do not store sensitive information (credentials) in your Terraform files.
+  Enter your `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, region, and output format which can be retrieved from the [AWS Console](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).
+
+:::caution Ownership of the created resources
+
+A user who creates resources in AWS will always retain administrative access to those resources, including any Kubernetes clusters created. It is recommended to create a dedicated [AWS IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html) for Terraform purposes, ensuring that the resources are managed and owned by that user.
+
+[Create access keys](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for the new IAM user via the console and export them as `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` variables to use with the AWS CLI and `eksctl`
 
 :::
 
-:::warning
+#### 1. Create an S3 bucket for Terraform state management
 
-A user who creates resources in AWS will therefore own these resources. In this particular case, the user will always have admin access to the Kubernetes cluster until the cluster is deleted.
+Before setting up Terraform, you need to create an S3 bucket that will store the state file. This is important for collaboration and to prevent issues like state file corruption.
 
-Therefore, it can make sense to create an extra [AWS IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html) which credentials are used for Terraform purposes.
+You can optionally set the region as an environment variable upfront to avoid repeating it in each command:
 
-:::
+```bash
+export AWS_REGION=<your-region>
+```
 
-### Cluster module
+Replace `<your-region>` with your chosen AWS region (e.g., `eu-central-1`).
 
-This module creates the basic layout that configures AWS access and Terraform.
+Now, follow these steps to create the S3 bucket with versioning enabled:
 
-The following will use [Terraform modules](https://developer.hashicorp.com/terraform/language/modules), which allows abstracting resources into reusable components.
+1. **Open your terminal** and ensure the AWS CLI is installed and configured.
 
-The [Camunda provided module](https://github.com/camunda/camunda-tf-eks-module/tree/2.5.0/modules/eks-cluster) is publicly available. It's advisable to review this module before usage.
+2. **Run the following command** to create the S3 bucket, using the previously set `AWS_REGION` environment variable:
 
-1. In the folder where your `config.tf` resides, create an additional `cluster.tf`.
-2. Paste the following content into the newly created `cluster.tf` file to make use of the provided module:
+   ```bash
+   aws s3api create-bucket --bucket my-eks-tf-state --region $AWS_REGION \
```

Comment:
I'd suggest moving the name into an environment variable as well, makes it easier to use all the other commands.

S3 bucket names are globally unique, meaning whatever is defined there needs to be adjusted anyway by the user.